packages/hoprd/src/index.ts

```diff
@@ -118,10 +108,8 @@ function parseHosts(): HoprOptions['hosts'] {
 
 async function generateNodeOptions(): Promise<HoprOptions> {
   let options: HoprOptions = {
-    bootstrapNode: argv.runAsBootstrap,
     createDbIfNotExist: argv.init,
     network: argv.network,
-    bootstrapServers: argv.runAsBootstrap ? [] : [...(await getBootstrapAddresses(argv.bootstrapServers)).values()],
```

Comment:
same here, we need some flag, that makes hoprd publishing its IP address

Reply:
`--announce` does this